CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+## Releases
+
+This extension is automatically built and released by the [`Build and Publish Vercel Deployment Extension`](./.github/workflows/ci.yml) CI workflow. The workflow invokes the [build-and-publish.js](./scripts/build-and-publish.js) script. This script requires a environment variable `AZURE_TOKEN`. This variable must be an [Azure DevOps Personal Access Token with the `Marketplace: Publish` permission](https://learn.microsoft.com/en-us/azure/devops/extend/publish/command-line?view=azure-devops#create-a-personal-access-token). The environment variable is passed to the script as a [GitHub Actions encrypted secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository).
+
+> The current `AZURE_TOKEN` secret was generated by @Ethan-Arrowood. It expires on June 24th, 2024.
+
+The automatic process runs on every new commit to the `main` branch. It runs the [tfx-cli](https://www.npmjs.com/package/tfx-cli), which is the official Node.js CLI for Azure DevOps. Source code and documentation for this can be found here [microsoft/tfs-cli](https://github.com/microsoft/tfs-cli) (yes the names do not match 🤷‍♂️). The `tfx-cli` will fail if the version number in [vss-extension.json](./vss-extension.json) has not been changed.
+
+To create a new release:
+- Either within the feature/fix PR, or afterwards, update the task versions in the respective task.json file(s).
+  - The versions follow [SemVer](https://semver.org/).
+    - Increment `Patch` for **Fixes**.
+    - Increment `Minor` for **Features**.
+    - Increment `Major` for **Breaking Changes**.
+      - Before merging Major changes, consult with the team to ensure proper documentation and update paths exist.
+    - If you are unsure how to version the changes, consult with the team.
+- After updating the version(s) for the modified task(s), update the extension version within [vss-extension.json](./vss-extension.json)
+  - This also follows [SemVer](https://semver.org/), but how it is updated is based on the task.json version changes.
+    - The extension version is incremented based on the greatest version change within the task.json changes.
+    - As an example, if one task receives a `Patch` update, and the other receives a `Minor` update, then the extension version should increment its `Minor` version (and subsequently reset the `Patch` value to `0`).
+    - If you are unsure, consult with the team before pushing version changes.
+- After modifying the version values, create and push a commit.
+- Once merged or pushed to `main`, the CI will automatically run, and if the extension version value has changed, the workflow will succeed and the new version will be published to Visual Studio Marketplace.
```

Comment:
```suggestion
- Once merged or pushed to `main`, the CI will automatically run, and if the extension version value has changed, the [workflow will succeed](https://github.com/vercel/vercel-azure-devops-extension/actions/workflows/ci.yml) and the new version will be published to Visual Studio Marketplace.
```

Reply:
Already link to the workflow in the first paragraph